client/src/reducers/user.js

```diff
@@ -1,4 +1,4 @@
-export default function UserReducer(state = { mentors: [], current: null }, action) {
+export default function UserReducer(state = { mentors: [], currentUser: null }, action) {
```

Comment:
idk about calling it currentUser. B/c to access it you'd have to do `this.props.user.currentUser` but it's simpler and makes sense to me to say `this.props.user.current`